common/thorhelper/CMakeLists.txt

```diff
@@ -110,6 +110,10 @@ include_directories (
          ${HPCC_SOURCE_DIR}/testing/unittests
          ${HPCC_SOURCE_DIR}/system/security/shared
          ${HPCC_SOURCE_DIR}/fs/dafsclient
+#ifdef _USE_PARQUET
+         ${HPCC_SOURCE_DIR}/esp/platform
```

Comment:
Why is the dependency on esp/platform needed?

Reply:
The esp/platform dependency is needed for tokenserialization.hpp.